### PR TITLE
[WIP] Create Passman script

### PR DESCRIPTION
**Pull request by enoch85**
cc @nextcloud/vm

### CODE REVIEW

**Comment by brantje** [comment]
Awesome! I plan to release 2.0 stable this week.

**Comment by enoch85** [comment]
@brantje Mind the tagging. This script automatically fetches the latest version from Git API so versions must be "v2.0.0" "v2.0.1" and so on, if that's ok with you?

**Comment by enoch85** [comment]
Note to self: curl -s https://api.github.com/repos/nextcloud/passman/releases/latest | grep 'tag_name' | cut -d\" -f4 | sed -e "s|v||g")

File: static/passman.sh [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
+then
+    echo
+    echo -e "\e[31mSorry, you are not root.\n\e[0mYou must type: \e[36msudo \e[0mbash $SCRIPTS/passman.sh"
+    echo
+    exit 1
+fi
+
+# Check if file is downloadable
+echo "Checking latest released version on the Passman download server and if it's possible to download..."
+curl -s wget -q $PASSVER_REPO/$PASSVER/$PASSVER_FILE > /dev/null
```
Comment (enoch85) [inline static/passman.sh]: Use pure curl instead.

**Pull request merged**